src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/LumberAxe.java

```diff
@@ -67,6 +67,10 @@ private ToolUseHandler onBlockBreak() {
     public ItemUseHandler onItemUse() {
         return e -> {
             if (e.getClickedBlock().isPresent()) {
+                if (e.getPlayer().isSneaking()) {
```

Comment:
Would this not be better combined with the previous if statement? Inverted of course.

Reply:
oh yeah, definitely, looked over it